pkgs/test/default.nix

```diff
@@ -22,6 +22,8 @@ with pkgs;
   cc-wrapper-libcxx-7 = callPackage ./cc-wrapper { stdenv = llvmPackages_7.libcxxStdenv; };
   stdenv-inputs = callPackage ./stdenv-inputs { };
 
+  haskell-shellFor = callPackage ./haskell-shellFor { };
```

Comment:
It looks like this file doesn't exist yet in this PR?  Was this an oversight?  Or is it still being written?

Reply:
@jmininger did we forget to commit??